lua/entities/gmt_instrument_base/init.lua

```diff
@@ -17,11 +18,18 @@ function ENT:Initialize()
 	if IsValid( phys ) then
 		phys:Wake()
 	end
+	
+	self:SetNWInt("CurrentInstrument",1)
 
 	self:InitializeAfter()
 	
 end
 
+net.Receive("ChangeInstrument",function()

```

Comment:
Not secure. check that the ent is valid, is a piano and that the player sending the message is sitting in the piano